RELEASING.md

```diff
@@ -1,12 +1,15 @@
 Releasing
 =========
 
+Note that these steps require pushing directly to the master branch on GitHub.  Be sure that other developers are not pushing changes concurrently.
+
  1. Change the version in `gradle.properties` to a non-SNAPSHOT version.
- 2. Update all README files with the new version.
+ 2. Update all README files (except `README-developers.md`) with the new release version.
  3. `git commit -am "Prepare for release X.Y.Z."` (where X.Y.Z is the new version)
```

Comment:
I like the commands to be copy-and-pasteable to the extent possible.  So I would suggest adding a step

export NEWVER=X.Y.Z (where X.Y.Z is the new version)

and then the subsequent steps can use that environment variable.